packages/amplify-console-integration-tests/src/profile-helper.ts

```diff
@@ -0,0 +1,101 @@
+import { getProfileName } from './util';
```

Comment:
Dont we have a similar method in amplify-e2e-tests. Lets have only one implementation which is a superset of features and move it to amplify-e2e-core

Reply:
The profile Name is different in the console-integration-tests, this profile will use the aws account specially set up for the Amplify Console related tests.

Reply:
we could make the helper function accept the profile name and regions then, instead of having multiple copies and making it harder to maintain